lib/src/ansi_code.dart

```diff
@@ -0,0 +1,278 @@
+// Copyright 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' as io;
+
+/// The type of code represented by [AnsiCode].
+class AnsiCodeType {
+  /// The name of the type.
```

Comment:
[nit] This comment isn't adding new information - others too - was this to satisfy a lint?